test/suites/dev/test-precompile/test-precompile-conviction-voting5.ts

```diff
@@ -21,18 +23,10 @@ describeSuite({
       );
       const alithAccount = await context.polkadotJs().query.system.account(ALITH_ADDRESS);
 
-      const rawTxn = await context.writePrecompile!({
-        precompileName: "ConvictionVoting",
-        functionName: "voteYes",
-        args: [proposalIndex, alithAccount.data.free.toBigInt() - 20n * 10n ** 18n, 1],
-        rawTxOnly: true,
-      });
-
-      await context.createBlock(
-        rawTxn,
-
-        { allowFailures: false }
-      );
+      let convictionVoting = new ConvictionVoting(context);
```

Comment:
Shadowing isn't supported in typescript so shouldn't work (im surprised the tests pass )

Reply:
they have different scopes.